libraries/helpers.rb

```diff
@@ -0,0 +1,18 @@
+module HabitatCookbook
+  module Helpers
+    def hab(*command)
+      begin
+        if defined?(shell_out_with_timeout!)
```

Comment:
this is leaving a huge landmine and you're using the side effect of the method being defined to select it.

really i think you're doing a lot of work to try to dry code out that doesn't need to be that dry:

```ruby
  def hab(*command)
    shell_out_with_timeout!(a_to_s("hab", *command))
  rescue  Errno::ENOENT
    Chef::Log.fatal("'hab' binary not installed, use the 'hab_install' resource to install it first")
  end
```

its fine to copypasta that around and avoid the use of a module completely.

if you want to centralize the error text you could drop it into Chef::Exceptions.